test/ls.js

```diff
@@ -144,7 +144,7 @@ assert.equal(result.indexOf('resources/ls/file1.js') > -1, true);
 
 // wildcard, should not do partial matches
 var result = shell.ls('resources/ls/*.j'); // shouldn't get .js
-assert.equal(shell.error(), null);
+assert.ok(shell.error());
```

Comment:
Not sure why this was the opposite before, but this seems like it's correct now. Good fix